src/Microsoft.DotNet.Interactive.Http.Tests/ParserTests.DynamicExpressions.cs

```diff
@@ -30,6 +30,25 @@ public void can_bind_datetime_with_custom_format()
             binding.Value.As<string>().Should().Be(currentTime.AddDays(1.0).ToString("dd-MM-yyyy").ToString());
         }
 
+        [Fact]
+        public void can_bind_datetime_uses_utcnow()
+        {
+            using var _ = new AssertionScope();

```

Comment:
The `AssertionScope` is not needed here. It's only something we typically do when there are multiple, non-interdependent assertions in the test, so that we can see the results of all of them even if an earlier one fails.

Reply:
Ah, just followed the pattern in the other tests - is it worth it to remove, or ok to leave?

Reply:
I'll go ahead and remove from all the unit tests that dont require it.